examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>"These are being trusted of not collaborating with each other" not necessarily! The exact term is colluding.</p><p>You can say that it is resistant to some level of collusion (we often consider that we have a <em>honest majority</em>), and can be<u> seen as</u> a <strong>trusted execution environment</strong> as long as thee isn't too much collusion. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Ok interesting, thanks! I'll change that.</p>